compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -666,8 +666,8 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 };
                 return Ok((discr, index));
             }
-            Variants::Multiple { tag, ref tag_encoding, tag_field, .. } => {
-                (tag, tag_encoding, tag_field)
+            Variants::Multiple { scalar, ref encoding, field, .. } => {
+                (scalar, encoding, field)
```

Comment:
'scalar', in the interpreter, usually refers to an `interpret::Scalar`. Please avoid using that name for things that have another type.

Reply:
Alright. But given that objects of this type shouldn't be referred to as `scalar`s... maybe `Scalar` is not a good name for the type?

Reply:
I consider its full name to be `abi::Scalar`.

My understanding is that the name makes more sense outside of the interpreter, but given that the interpreter has `interpret::Scalar` we have a somewhat unfortunate conflict.